### DISCUSSION TITLE
scripts/ccutils.sh: line 25: jq: command not found

### DISCUSSION THREAD

**Discussion by mock123456** [hyperledger/fabric#3636] [Q&A]
When i keyboard the ./fabcar command to test the network,i found a error.I receive this message:

scripts/ccutils.sh: line 25: jq: command not found

res=1
Query installed on peer0.org1 has failed
Deploying chaincode failed

**Comment by xyrold** [comment]
hi, did you fix yours? i'm having same exact error...

**Comment by mock123456** [comment]
you can try&nbsp; to keyboard pip install jq command in terminal.



---Original---
From: ***@***.***&gt;
Date: Thu, Oct 13, 2022 10:21 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hyperledger/fabric] scripts/ccutils.sh: line 25: jq: command notfound (Discussion #3636)




 
hi, did you fix yours? i'm having same exact error...

**Reply by xyrold** [reply]
great! you saved me a day... thanks a lot!

btw, i'm using ubuntu on windows.... "sudo apt-get install jq"
